Document/0x06h-Testing-Platform-Interaction.md

```diff
@@ -569,23 +569,18 @@ In addition:
 - No long-running background tasks are allowed but uploads or downloads can be initiated.
 - App extensions cannot access the camera or microphone on an iOS device (except for iMessage app extensions).
 
-#### UIPasteboard
+#### Pasteboard
 
-When typing data into input fields, the clipboard can be used to copy in data. The clipboard is accessible system-wide and is therefore shared by apps. This sharing can be misused by malicious apps to get sensitive data that has been stored in the clipboard.
+Apps can access the iOS pasteboard using the [`UIPasteboard`](https://developer.apple.com/documentation/uikit/uipasteboard) API. It allows apps to share data either within the app or across apps. However, the systemwide nature of the general pasteboard introduces privacy and security concerns, particularly when sensitive data is copied programatically without user interaction.
 
-When using an app you should be aware that other apps might be reading the clipboard continuously, as the [Facebook app](https://www.thedailybeast.com/facebook-is-spying-on-your-clipboard "Facebook Is Spying On Your Clipboard") did. Before iOS 9, a malicious app might monitor the pasteboard in the background while periodically retrieving `[UIPasteboard generalPasteboard].string`. As of iOS 9, pasteboard content is accessible to apps in the foreground only, which reduces the attack surface of password sniffing from the clipboard dramatically. Still, copy-pasting passwords is a security risk you should be aware of, but also cannot be solved by an app.
+There are two types of pasteboards:
 
-- Preventing pasting into input fields of an app, does not prevent that a user will copy sensitive information anyway. Since the information has already been copied before the user notices that it's not possible to paste it in, a malicious app has already sniffed the clipboard.
-- If pasting is disabled on password fields users might even choose weaker passwords that they can remember and they cannot use password managers anymore, which would contradict the original intention of making the app more secure.
+- **General pasteboard (`UIPasteboard.general`)**: Shared across all foreground apps and, with Universal Clipboard, potentially across Apple devices. Persistent by default across device restarts and app reinstalls unless cleared. Requires user interaction for access as of iOS 16.
+- **Custom or named pasteboards (`UIPasteboard(name:create:)`, `UIPasteboard.withUniqueName()`)**: App-specific or team-specific. Non-persistent by default since iOS 10 and restricted to the creating app or apps from the same team ID. Apple discourages using persistent custom pasteboards and recommends [using App Groups](https://developer.apple.com/documentation/Xcode/configuring-app-groups) for sharing data between apps of the same developer.
```

Comment:
```suggestion
- **Custom or Named Pasteboards (`UIPasteboard(name:create:)` and `UIPasteboard.withUniqueName()`)**: These are [private pasteboards](https://developer.apple.com/library/archive/documentation/StringsTextFonts/Conceptual/TextAndWebiPhoneOS/UsingCopy%2CCut%2CandPasteOperations/UsingCopy%2CCut%2CandPasteOperations.html) that are app- or team-specific, i.e., restricted to the app that created them or other apps from the same team ID. They are non-persistent by default since iOS 10 (deleted upon app termination and system reboot). Apple discourages the use of persistent custom pasteboards and recommends [using App Groups](https://developer.apple.com/documentation/Xcode/configuring-app-groups) for sharing data between apps of the same developer.
```